resources/views/livewire/member/member.blade.php

```diff
@@ -0,0 +1,191 @@
+                        class="bg-[#f1d07c] font-semibold rounded-md text-base px-7 py-2.5 me-2 mb-2">INVITE</button>
+                </div>
+            </div>
+            <div class="border rounded-lg p-7 bg-white h-fit">
+                <div class="flex items-center gap-4 mb-3">
+                    <img class="w-10 h-10 rounded-full"
+                        src="https://images.unsplash.com/photo-1633332755192-727a05c4013d?ixlib=rb-1.2.1&amp;ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&amp;auto=format&amp;fit=crop&amp;w=1480&amp;q=80"
+                        alt="">
+                    <div class="font-medium dark:text-white">
+                        <div>Jese Leos</div>
+                        <div class="text-sm font-semibold text-gray-500 dark:text-gray-400">
+                            @genevive-dikeanowe-5859
+                        </div>
+                    </div>
+                </div>
+                <div class="flex flex-col">
+                    <p class="text-gray-500 dark:text-gray-400 mb-3">Track work across the enterprise through an open,
+                        collaborative
+                        platform. Link issues across Jira and ingest data from other software development tools, so your
+                        IT
+                        support.
+                    </p>
+                    <p class="text-gray-500 dark:text-gray-400 mb-2 flex gap-2">
+                        <svg class="w-5 h-5 text-green-800 dark:text-white" aria-hidden="true"
+                            xmlns="http://www.w3.org/2000/svg" fill="currentColor" viewBox="0 0 24 24">
+                            <path fill-rule="evenodd"
+                                d="M2 12a10 10 0 1 1 20 0 10 10 0 0 1-20 0Zm13.7-1.3a1 1 0 0 0-1.4-1.4L11 12.6l-1.8-1.8a1 1 0 0 0-1.4 1.4l2.5 2.5c.4.4 1 .4 1.4 0l4-4Z"
+                                clip-rule="evenodd" />
+                        </svg>
+                        <span>Online now</span>
+                    </p>
+                    <p class="text-gray-500 dark:text-gray-400 mb-2 flex gap-2">
+                        <svg class="w-5 h-5 text-gray-800 dark:text-white" aria-hidden="true"
+                            xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24">
+                            <path stroke="currentColor" stroke-linecap="round" stroke-linejoin="round" stroke-width="2"
+                                d="M4 10h16m-8-3V4M7 7V4m10 3V4M5 20h14c.6 0 1-.4 1-1V7c0-.6-.4-1-1-1H5a1 1 0 0 0-1 1v12c0 .6.4 1 1 1Zm3-7h0v0h0v0Zm4 0h0v0h0v0Zm4 0h0v0h0v0Zm-8 4h0v0h0v0Zm4 0h0v0h0v0Zm4 0h0v0h0v0Z" />
+                        </svg> <span>Joined March 04, 2024</span>
+                    </p>
+
+                </div>
+                <hr class="h-px my-8 bg-gray-200 border-0 dark:bg-gray-700">
+            </div>
+
+        </div>
+
+        <div class="w-1/4">
+            <div class="border rounded-lg bg-white overflow-hidden">
```

Comment:
use the community-info component for this. 

` <livewire:communities.components.community-info :community="$community"/>`